semantic-conventions/syntax.md

```diff
@@ -40,10 +40,15 @@ here in `syntax.md` should be considered more authoritative though. Please keep
 All attributes are lower case.
 
 ```ebnf
-groups ::= semconv
-       | semconv groups
+groups ::= group
```

Comment:
There's `group(s)` many times here and it makes me very confused 😅. Not sure what but I think it would be best if we name it differently. Maybe `convention`? And also couldn't `namespace` be `attribute_registry`? Something like:

```ebnf
groups ::= convention
       | convention groups

convention::= semconv
      | attribute_registry

attribute_registry ::= id brief [note] prefix [stability] [deprecated] attribute_definitions
```